src/scss/nav.component.scss

```diff
@@ -84,6 +84,10 @@ nav {
         }
       }
     }
+    .docs-link {
+      font-size: 1rem;
+      margin: 0.4rem;
+    }
```

Comment:
Can't see class `docs-link` used anywhere. Is it?

Reply:
Got this cleaned up. Good catch! :)